packages/oscal-react-library/src/components/OSCALCatalogGroup.tsx

```diff
@@ -159,96 +145,47 @@ const CollapsibleListItem: React.FC<CollapsibleListItemProps> = (props) => {
     previousHandledFragment,
     setPreviousHandledFragment,
   ]);
-
   return (
-    <StyledListItemPaper>
-      <StyledListItem onClick={handleClick}>
-        <ListItemText primary={itemText} />
-        {isOpen ? <ExpandLess /> : <ExpandMore />}
-      </StyledListItem>
-      <Collapse
-        in={isOpen}
-        timeout="auto"
-        onEntered={() => !group && setListItemOpened(true)}
-        unmountOnExit
-      >
-        <StyledControlDescriptionWrapper>{children}</StyledControlDescriptionWrapper>
-      </Collapse>
-    </StyledListItemPaper>
+    <Accordion
+      expanded={isOpen}
+      onChange={handleClick}
+      TransitionProps={{
+        onEntered: () => !group && setListItemOpened(true),
+        unmountOnExit: true,
+        timeout: 0,
+      }}
+      disabled={false && props.isEmpty}
```

Comment:
Thinking of just never disabling... the styling is weird (see https://mui.com/material-ui/react-accordion/#basic-accordion). If you want this, please provide a good style for disabled accordions. Otherwise, I think having a small expand isn't that bad.

I do like that this makes `disabled` totally distinct from `withdrawn`